### PR TITLE
move decodeMoves and move2gtpvertex shared functions to /utils

### PR DESCRIPTION
**Pull request by wonderingabout**
tested to work, ready to merge
game works end to end on ogs

partially adresses #281

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+
+const { decodeMoves }    = require("./utils/decodeMoves");
+const { gtpchar2num }    = require("./utils/gtpchar2num");
+const { move2gtpvertex } = require("./utils/move2gtpvertex");
 
 const child_process = require('child_process');
-const console = require('./console').console;
-const config = require('./config');
-const { Pv } = require('./pv');
+const console       = require('./console').console;
```
Comment (roy7) [inline bot.js:10]: Is the extra whitespace on these 3 lines intentional?

Comment (wonderingabout) [reply bot.js:10]: yes, i wanted to align the equal with the line just above to make it more readable
should i revert it to how it was?

**Comment by roy7** [comment]
I'll just merge it, but I don't usually like to format code like that. It's just a style choice. If @Dorus objects we can easily swap it back.

**Comment by Dorus** [comment]
Yes i agree, ugly formatting

**Comment by wonderingabout** [comment]
fixed in https://github.com/online-go/gtp2ogs/pull/309

**Comment by wonderingabout** [comment]
i think we were a bit too quick on the merge here but hopefully all is good now :sweat_smile:

**Pull request merged**